info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:38:59.827Z"}
info: Connected to the database. {"timestamp":"2023-01-13T03:39:08.901Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:41:32.165Z"}
info: Connected to the database. {"timestamp":"2023-01-13T03:41:40.962Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:42:59.753Z"}
info: Connected to the database. {"timestamp":"2023-01-13T03:43:08.812Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:44:15.499Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:45:00.442Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:45:53.333Z"}
info: Connected to the database. {"timestamp":"2023-01-13T03:46:02.322Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:53:13.073Z"}
info: Connected to the database. {"timestamp":"2023-01-13T03:53:22.374Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T03:59:53.372Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:00:11.015Z"}
info: Connected to the database. {"timestamp":"2023-01-13T04:00:16.243Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:00:33.888Z"}
info: Connected to the database. {"timestamp":"2023-01-13T04:00:39.053Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:01:23.203Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:01:40.984Z"}
info: Connected to the database. {"timestamp":"2023-01-13T04:01:46.271Z"}
error: 
    status - 500 
    message - ValidationError: "tuser_id" is required 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T04:03:18.710Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:04:06.064Z"}
info: Connected to the database. {"timestamp":"2023-01-13T04:04:09.940Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:04:31.008Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:04:48.929Z"}
info: Connected to the database. {"timestamp":"2023-01-13T04:04:52.868Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T04:05:48.896Z"}
info: Connected to the database. {"timestamp":"2023-01-13T04:05:52.672Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/notes/add 
      method - POST 
      IP - ::1
     {"timestamp":"2023-01-13T04:06:04.384Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:28:29)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at userAuth (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:16:22)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at newnoteValidator (C:\Users\Yatendra Pratap\Fundoo_Notes\src\validators\/note.validator.js:15:5) 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T04:06:58.210Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:28:29)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at userAuth (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:16:22)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at newnoteValidator (C:\Users\Yatendra Pratap\Fundoo_Notes\src\validators\/note.validator.js:15:5) 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T04:07:12.572Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T05:42:22.556Z"}
error: Could not connect to the database. querySrv ECONNREFUSED _mongodb._tcp.demo.aoloa.mongodb.net {"code":"ECONNREFUSED","hostname":"_mongodb._tcp.demo.aoloa.mongodb.net","stack":"Error: querySrv ECONNREFUSED _mongodb._tcp.demo.aoloa.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/callback_resolver:47:19)","syscall":"querySrv","timestamp":"2023-01-13T05:42:22.616Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T05:42:33.680Z"}
error: Could not connect to the database. querySrv ECONNREFUSED _mongodb._tcp.demo.aoloa.mongodb.net {"code":"ECONNREFUSED","hostname":"_mongodb._tcp.demo.aoloa.mongodb.net","stack":"Error: querySrv ECONNREFUSED _mongodb._tcp.demo.aoloa.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/callback_resolver:47:19)","syscall":"querySrv","timestamp":"2023-01-13T05:42:33.696Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T05:42:49.962Z"}
error: Could not connect to the database. querySrv ECONNREFUSED _mongodb._tcp.demo.aoloa.mongodb.net {"code":"ECONNREFUSED","hostname":"_mongodb._tcp.demo.aoloa.mongodb.net","stack":"Error: querySrv ECONNREFUSED _mongodb._tcp.demo.aoloa.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/callback_resolver:47:19)","syscall":"querySrv","timestamp":"2023-01-13T05:42:50.001Z"}
error: 
    status - 500 
    message - MongooseError: Operation `notes.find()` buffering timed out after 10000ms
    at Timeout.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:564:17)
    at processTimers (node:internal/timers:507:7) 
    url - /api/v1/notes 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T05:44:54.152Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T05:49:00.355Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T05:49:10.284Z"}
info: Connected to the database. {"timestamp":"2023-01-13T05:49:20.668Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:20:52.952Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:21:05.383Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:21:21.230Z"}
info: Connected to the database. {"timestamp":"2023-01-13T06:21:25.652Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:31:42.480Z"}
info: Connected to the database. {"timestamp":"2023-01-13T06:31:55.434Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:33:15.796Z"}
info: Connected to the database. {"timestamp":"2023-01-13T06:33:28.795Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:39:32.489Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:39:40.397Z"}
info: Connected to the database. {"timestamp":"2023-01-13T06:39:47.071Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:41:52.771Z"}
info: Connected to the database. {"timestamp":"2023-01-13T06:42:02.488Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T06:52:44.033Z"}
error: Could not connect to the database. queryTxt ETIMEOUT demo.aoloa.mongodb.net {"code":"ETIMEOUT","hostname":"demo.aoloa.mongodb.net","stack":"Error: queryTxt ETIMEOUT demo.aoloa.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/callback_resolver:47:19)","syscall":"queryTxt","timestamp":"2023-01-13T07:16:37.057Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:36:36.689Z"}
info: Connected to the database. {"timestamp":"2023-01-13T07:36:46.095Z"}
error: 
    status - 500 
    message - ValidationError: "user_id" is required 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T07:48:18.644Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:49:13.961Z"}
info: Connected to the database. {"timestamp":"2023-01-13T07:49:23.534Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:50:47.029Z"}
info: Connected to the database. {"timestamp":"2023-01-13T07:50:51.220Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:51:00.933Z"}
info: Connected to the database. {"timestamp":"2023-01-13T07:51:06.311Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:51:57.968Z"}
info: Connected to the database. {"timestamp":"2023-01-13T07:52:03.371Z"}
error: 
    status - 500 
    message - ValidationError: "user_id" is required 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T07:52:20.920Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:53:53.779Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:54:10.071Z"}
info: Connected to the database. {"timestamp":"2023-01-13T07:54:14.388Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:28:29)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at userAuth (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:16:22)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at newnoteValidator (C:\Users\Yatendra Pratap\Fundoo_Notes\src\validators\/note.validator.js:14:5)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:204:9)
    at invokeCallback (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:513:28)
    at IncomingMessage.emit (node:domain:489:12) 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T07:54:19.079Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:58:31.950Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T07:58:40.302Z"}
info: Connected to the database. {"timestamp":"2023-01-13T07:58:49.602Z"}
error: 
    status - 500 
    message - ValidationError: "user_id" is required 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T07:58:58.209Z"}
error: 
    status - 500 
    message - ValidationError: "user_id" length must be at least  characters long 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T08:00:13.663Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:00:58.524Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:01:07.173Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:28:29)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at userAuth (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:16:22)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at newnoteValidator (C:\Users\Yatendra Pratap\Fundoo_Notes\src\validators\/note.validator.js:15:5)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:204:9)
    at invokeCallback (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:513:28)
    at IncomingMessage.emit (node:domain:489:12) 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T08:01:11.621Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:11:01.776Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:11:11.050Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:11:24.831Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:11:33.165Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:11:39.516Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:11:58.613Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:12:13.537Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:12:18.835Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:12:33.778Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:12:38.986Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:13:44.560Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:13:49.843Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:13:56.921Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:14:38.223Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:14:41.888Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:14:51.257Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:14:56.542Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:15:24.966Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:15:30.217Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:15:47.894Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:15:52.306Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:16:52.399Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:16:57.823Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:17:34.122Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:17:39.496Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:18:11.210Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:18:15.706Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "rahul123@gmail.com" (type string) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/rahul123@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:18:41.573Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "rahul123@gmail.com" (type string) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/rahul123@gmail.com 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:19:52.585Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:25:21.450Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:25:30.071Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "all" (type string) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:25:37.517Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:26:26.572Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:26:35.230Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:27:50.967Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:27:56.175Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got rahul123@gmail.com
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee6$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:44:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:43:24)
    at _callee3$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:34:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:32:24)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:28:12.233Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:31:23.090Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got rahul123@gmail.com
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee6$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:44:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:43:24)
    at _callee3$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:34:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:32:24)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:31:23.903Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:31:31.818Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got rahul123@gmail.com
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee6$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:44:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:43:24)
    at _callee3$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:34:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:32:24)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:31:39.788Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:32:24.412Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:32:27.628Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got rahul123@gmail.com
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee6$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:44:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:43:24)
    at _callee3$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:34:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:32:24)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:32:30.209Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:33:26.458Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:33:57.108Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:34:06.059Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:34:09.570Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got rahul123@gmail.com
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee6$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:44:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:43:24)
    at _callee3$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:35:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:32:24)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:34:30.651Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:35:33.149Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:35:42.016Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got rahul123@gmail.com
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee6$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:45:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:43:24)
    at _callee3$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:35:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getallnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:32:24)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T08:35:44.730Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:37:46.014Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:37:51.320Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:38:03.062Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:38:08.529Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:39:01.944Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:39:08.006Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:39:24.983Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:39:30.238Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:39:46.053Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:39:50.839Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:40:05.271Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:40:10.517Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T08:41:00.936Z"}
info: Connected to the database. {"timestamp":"2023-01-13T08:41:09.725Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:13:23.262Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:13:33.440Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:13:44.044Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:13:51.227Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:14:15.589Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:14:24.596Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:16:08.007Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:16:16.745Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:16:43.702Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:16:59.362Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:17:09.552Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:17:22.728Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:17:26.713Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "add" (type string) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/add 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T09:19:55.273Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:24:28.889Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:24:37.804Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:26:27.239Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:26:32.544Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:31:11.014Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:31:14.522Z"}
error: 
    status - 500 
    message - ValidationError: "firstname" is required 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T09:31:23.801Z"}
error: 
    status - 500 
    message - ValidationError: "firstname" is required 
    url - /api/v1/users//registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T09:33:03.558Z"}
error: 
    status - 500 
    message - ValidationError: "firstname" is required 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T09:35:58.199Z"}
error: 
    status - 500 
    message - ValidationError: "firstname" is required 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T09:37:06.828Z"}
error: 
    status - 500 
    message - ValidationError: "user_id" is required 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T09:40:55.678Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:46:35.449Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:46:44.732Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:47:20.753Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:47:28.499Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:47:33.196Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:48:44.322Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:48:52.273Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:48:57.257Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:49:22.971Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:49:28.201Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:49:38.337Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T09:49:43.809Z"}
info: Connected to the database. {"timestamp":"2023-01-13T09:49:48.786Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T11:41:50.226Z"}
info: Connected to the database. {"timestamp":"2023-01-13T11:42:00.243Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T11:49:01.457Z"}
info: Connected to the database. {"timestamp":"2023-01-13T11:49:10.315Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:04:32.331Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:04:41.478Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:06:36.927Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:06:47.926Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:07:27.743Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:07:36.990Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:08:25.327Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:08:30.997Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:09:10.351Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:09:15.734Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:12:11.838Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:12:29.980Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:12:39.511Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:12:47.679Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:12:56.603Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:13:17.488Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:13:21.794Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:14:02.432Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:14:09.032Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:18:47.810Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:18:56.507Z"}
error: 
    status - 500 
    message - ValidationError: "user_id" is required 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T12:20:54.556Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:23:31.072Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:23:41.933Z"}
error: 
    status - 500 
    message - ValidationError: "user_id" is required 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T12:27:05.403Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:30:02.112Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:30:11.437Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:30:34.579Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:30:42.023Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:30:45.662Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:31:05.416Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:31:10.943Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:31:57.582Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:32:02.838Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:32:12.617Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:32:17.781Z"}
error: 
    status - 500 
    message - ReferenceError: error is not defined
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:12)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T12:32:34.598Z"}
error: 
    status - 500 
    message - ReferenceError: error is not defined
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:12)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/add 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T12:33:07.650Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:33:43.867Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:33:52.474Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:34:21.685Z"}
error: 
      status - 400
      message - Invalid credential 
      url - /api/v1/notes/add 
      method - POST 
      IP - ::1
     {"timestamp":"2023-01-13T12:34:26.380Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:34:27.115Z"}
error: 
    status - 500 
    message - SyntaxError: Expected double-quoted property name in JSON at position 57
    at JSON.parse (<anonymous>)
    at parse (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:204:9)
    at invokeCallback (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:513:28)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/notes/all 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-13T12:34:51.784Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:38:09.556Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:38:21.305Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got 63c150e89ecefa41403dc1ec
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:11:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:10:21)
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:21:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:19:21)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c150e89ecefa41403dc1ec 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T12:39:27.005Z"}
error: 
    status - 500 
    message - SyntaxError: Expected double-quoted property name in JSON at position 63
    at JSON.parse (<anonymous>)
    at parse (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:204:9)
    at invokeCallback (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:513:28)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/notes/63c150e89ecefa41403dc1ec 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T12:39:38.380Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got 63c150e89ecefa41403dc1ec
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:11:27)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:10:21)
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:21:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at getnote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:19:21)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c150e89ecefa41403dc1ec 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T12:39:42.918Z"}
error: 
      status - 409
      message - Invalid credential 
      url - /api/v1/notes/all 
      method - GET 
      IP - ::1
     {"timestamp":"2023-01-13T12:41:00.268Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:42:01.640Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:42:10.302Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:42:58.219Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:43:09.643Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:43:50.935Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:43:56.002Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:45:29.958Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:46:42.212Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:46:51.035Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:47:35.944Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:47:44.813Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:47:55.140Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:48:00.488Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:48:25.109Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:48:31.247Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:48:40.793Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:48:46.080Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:49:01.675Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:49:10.255Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:49:46.478Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:49:55.947Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:49:59.904Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:50:56.357Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:51:08.055Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:51:13.221Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:51:29.796Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:51:39.220Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:52:19.221Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:52:28.983Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'find')
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:17:30)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c14f6fa5dec6354cd1d309 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T12:55:01.304Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:55:47.435Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:55:56.326Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'find')
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:17:30)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c14f6fa5dec6354cd1d309 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T12:56:07.998Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:56:58.190Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:57:09.508Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:57:14.735Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got 63c14f6fa5dec6354cd1d309
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:17:31)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c14f6fa5dec6354cd1d309 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T12:57:23.043Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:58:23.228Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:58:28.489Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:59:01.132Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:59:07.743Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T12:59:22.172Z"}
info: Connected to the database. {"timestamp":"2023-01-13T12:59:32.622Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got 63c14f6fa5dec6354cd1d309
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:17:31)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c14f6fa5dec6354cd1d309 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T12:59:36.475Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:00:41.251Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:00:54.045Z"}
error: 
    status - 500 
    message - ObjectParameterError: Parameter "filter" to find() must be an object, got 63c14f6fa5dec6354cd1d309
    at model.Query.Query.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:2098:16)
    at Function.find (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\model.js:2157:13)
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:18:31)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c14f6fa5dec6354cd1d309 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T13:02:31.656Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:04:51.381Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:05:00.352Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:07:32.476Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:07:41.577Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:08:01.652Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:08:10.446Z"}
error: 
      status - 401
      message - Invalid credential 
      url - /api/v1/notes/63c14f6fa5dec6354cd1d309 
      method - GET 
      IP - ::1
     {"timestamp":"2023-01-13T13:08:29.806Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "63c14f6fa5dec6354cd1d30" (type string) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c14f6fa5dec6354cd1d30 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T13:08:41.945Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:09:23.792Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:09:40.514Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:09:49.419Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:10:01.429Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "63c14f6fa5dec6354cd1d30" (type string) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c14f6fa5dec6354cd1d30 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T13:10:08.707Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:11:18.194Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:11:24.195Z"}
error: 
    status - 500 
    message - ReferenceError: error is not defined
    at _callee2$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:21:23)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c1106a862cf415b0295b39 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T13:12:04.335Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:13:13.591Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:13:22.637Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:18:06.268Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:18:15.066Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:19:54.496Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:19:59.053Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:20:40.006Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:20:45.324Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:21:41.554Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:21:52.799Z"}
error: 
    status - 500 
    message - ReferenceError: error is not defined
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:9:16)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/all 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-13T13:21:57.907Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:22:52.381Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:23:17.100Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:23:26.551Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:23:47.547Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:24:10.809Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:24:24.735Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:24:41.089Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:24:46.737Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:28:44.585Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:28:53.696Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:29:04.698Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:29:13.100Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:31:08.009Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:31:15.962Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:31:28.535Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:31:35.982Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:31:50.500Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:31:58.723Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:32:07.830Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:32:42.228Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:32:49.461Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:32:57.509Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:33:17.762Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:33:35.318Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:33:48.307Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:33:57.647Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:34:06.115Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:34:11.340Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:34:32.251Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:34:37.380Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:35:54.183Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:36:00.354Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:36:35.911Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:36:44.721Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:37:20.493Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:37:24.095Z"}
error: 
      status - 401
      message - Invalid credential 
      url - /api/v1/notes/63c15eee9b40064734a0512a 
      method - GET 
      IP - ::1
     {"timestamp":"2023-01-13T13:39:21.403Z"}
error: 
    status - 500 
    message - ReferenceError: body is not defined
    at _callee4$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:47:7)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c15eee9b40064734a0512a 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-01-13T13:40:27.999Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:43:57.535Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:44:04.699Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:44:14.693Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:44:20.657Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:44:31.237Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:44:48.661Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:45:00.946Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:45:08.970Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:45:16.684Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:45:30.798Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:45:34.913Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:45:48.331Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:46:02.630Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:46:07.874Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:46:36.069Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:46:46.086Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:46:51.695Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ noteid: '63c1612805e6ed1370e2af72' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c1612805e6ed1370e2af72 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-01-13T13:50:10.448Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:52:24.539Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:52:32.119Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:52:42.230Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:53:00.829Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:53:16.810Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:53:23.673Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ noteid: '63c1612805e6ed1370e2af72' }" (type Object) at path "_id" for model "Note"
    at model.Query.exec (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c1612805e6ed1370e2af72 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-01-13T13:54:01.268Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:55:03.044Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:55:10.815Z"}
error: 
    status - 500 
    message - ReferenceError: noteid is not defined
    at _callee4$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:40:33)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.updatenote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\services\/note.service.js:39:24)
    at _callee4$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:51:36)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at updatenote (C:\Users\Yatendra Pratap\Fundoo_Notes\src\controllers\/note.controller.js:49:24)
    at Layer.handle [as handle_request] (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (C:\Users\Yatendra Pratap\Fundoo_Notes\src\middlewares\/auth.middleware.js:31:5)
    at tryCatch (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Yatendra Pratap\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63c1612805e6ed1370e2af72 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-01-13T13:55:15.324Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:55:23.028Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:55:38.884Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:55:53.139Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:56:28.630Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:56:34.674Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:56:43.049Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:56:55.115Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:57:18.072Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:57:24.818Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T13:57:37.119Z"}
info: Connected to the database. {"timestamp":"2023-01-13T13:57:41.621Z"}
info: Server started at http://localhost:5454/api/v1/ {"timestamp":"2023-01-13T14:02:02.280Z"}
